lib/elixir/src/elixir_erl.erl

```diff
@@ -531,6 +531,9 @@ get_type_docs(Set, Types) ->
 signature_to_binary(_Module, Name, _Signature) when Name == '__aliases__'; Name == '__block__' ->
   <<(atom_to_binary(Name, utf8))/binary, "(args)">>;
 
+signature_to_binary(_Module, Name, _Signature) when Name == fn -> 
+  <<"fn [clauses] end">>;
```

Comment:
I think we can do `fn (clauses) end` or `fn ... end` or simply `fn`.

Reply:
Would like make sense to do something like this `atom_to_binary(Name, utf8);` then?

Reply:
In this case we know that the Name is atom, so we can skip the indirection. So we should rather write it like this:

```erlang
signature_to_binary(_Module, fn, _Signature) -> 
  <<"fn">>;
```

Reply:
Fixed